Lee Lichtsinn / React-UI-Components

leelichtsinn opened the pull request:
Yarn installed dependencies
@47analogy

leelichtsinn committed:
yarn installed dependencies

leelichtsinn committed:
added structure to HeaderContainer

leelichtsinn committed:
added import statements for ImageThumbnail and HeaderTitle

leelichtsinn committed:
added export statement to HeaderContainer

leelichtsinn committed:
added src to ImageThumbnail

leelichtsinn committed:
added HeaderTitle content

leelichtsinn committed:
Added import statement for HeaderContainer
Replaced App content with HeaderContainer

leelichtsinn committed:
Corrected import file paths for HeaderTitle and ImageThumbnail

leelichtsinn committed:
Added alt property to img tag

leelichtsinn committed:
Added content to HeaderContent

leelichtsinn committed:
added css classes to components

leelichtsinn committed:
added import for CardBanner
added css class name
added reactjs.org url

leelichtsinn committed:
added export statement

leelichtsinn committed:
added import for CardContent
added React.js logo href

leelichtsinn commented:
Built HeaderContainer component
Built HeaderTitle Component
Built ImageThumbnail component
Built HeaderContent component
Built CardContainer component
Built CardBanner component
Built CardContent component
Added all imports needed
Added components to App.js
@47analogy

leelichtsinn committed:
added content to CardContent
added CardContent import

leelichtsinn committed:
added base styles to App.js div

leelichtsinn committed:
refactored jsx layout

leelichtsinn committed:
added styles to HeaderContainer
added styles to ImageThumbnail
added styles to HeaderContent
added styles to HeaderTitle
added styles to HeaderTitle h1 and p

47analogy commented:
All content renders on the screen. Correct syntax and all child components imported correctly into the parent components. Great job, just a matter of styling the social card layout in the respective css folders (Card.css and Header.css).

leelichtsinn committed:
added styles to CardContainer
added styles to CardBanner

leelichtsinn committed:
added border-radius and border to CardContainer
refactored CardBanner css
added height to CardBanner img
added styles to CardContent

leelichtsinn committed:
installed dependencies

leelichtsinn committed:
added NumberButton functional component

leelichtsinn committed:
added NumberButton import
added NumberButton component
added buttonStyle and text props to NumberComponent

leelichtsinn committed:
added jsx content passing props

leelichtsinn committed:
refactored CardContainer text-decoration and text color

leelichtsinn committed:
added flex and width styles to App.css

leelichtsinn commented:
Added dependencies
Added NumberButton functional component
Added basic styles to App.css
@47analogy

47analogy commented:
For the calculator project, it looks like you are off to a good start: You are making use of reusable components and passing down the style and text as props. Looks like you just have to do the same for CalculatorDisplay.js and ActionButton.js for the appropriate content. Then layout the components according to the design file. Do you have any questions on how to proceed?

bikkimahato commented on Day-I-Social-Card/social-card/src/components/CardComponents/Card.css:
```diff
@@ -1 +1,54 @@
 /* All styles for your Card Components live here */
+.card-container {
+  display: flex;
+  /* flex-direction: column; */
```
Commented code should not be pushed to GitHub.
